src/Service/Backup/BackupInterface.php

```diff
@@ -12,21 +20,30 @@
  */
 interface BackupInterface
 {
+    /**
+     * @return string
```

Comment:
I really think that phpdocs which only duplicates the typehints are not meaningful since php 7